xml/article_setup.xml

```diff
@@ -1728,10 +1731,8 @@ noop deadline [cfq]</screen>
      More detailed information about the real-time Linux development and an
      introduction how to write a real-time application can be found in the
      real-time Linux community Wiki.
-     <link xlink:href="http://rt.wiki.kernel.org"/>,
-     <link
-            xlink:href="http://rt.wiki.kernel.org/index.php/HOWTO:_Build_an_RT-application"
-          />
+     <link xlink:href="http://rt.wiki.kernel.org"/>, <link xlink:href=
+     "http://rt.wiki.kernel.org/index.php/HOWTO:_Build_an_RT-application"/>
```

Comment:
* You can upgrade both links to HTTPS
* The second link has not been updated in any meaningful way since 2014. Not sure if that's a good sign. But I guess that's out of scope here.